contents/docs/tutorials/actions.md

```diff
@@ -0,0 +1,196 @@
+
+> **Note:** Actions also work retroactively, meaning that they also tag events that happened in the past, not only events that happened after the action was created.
+
+### Creating Actions
+
+There are many ways to create actions in PostHog. These are:
+
+**1. Through our toolbar (recommended for actions based on frontend elements)**
+
+To learn how to create actions using our toolbar, you can visit our [dedicated tutorial](/docs/tutorials/toolbar/#creating-actions).
+
+**2. From the events table**
+
+To create actions from the events table, just click on the plus sign on the left of any event to reveal a button allowing you to create an action from that specific event. You will also be able to customize the action further in the following steps (such as add another event). 
+
+![Action from event image](../../images/tutorials/actions/action-from-event.png)
+<small class='centered'>_Creating actions from the events table_</small>
+
+
+**3. From the 'Actions' page**
+
+Lastly, you can of course create actions from the 'Actions' page under 'Events':
+
+![Actions page image](../../images/features/actions/actions-page.png)
+
+
+**Action Options**
+
+Visit our [dedicated actions page](/docs/features/actions/) for a comprehensive explanation of all the options available when creating an action.
+
+### Upgrading Your Analytics Game with Custom Events
+
+Now that we've learned how to sort through events, let's go through how to create events that PostHog doesn't automatically capture for you.
+
+Custom events can be created from any of our [libraries](/docs/integrations), as well as our [API](/docs/api/overview). They can be triggered from both the backend and the frontend. 
+
+**Example: Custom events from your frontend**
+
+Frontend JS is heavily event-based by nature, making it a simple paradigm to pair up JavaScript events with PostHog events.
+
+For example, to trigger an event when an element is hovered, you can do:
+
+```js
+// Vanilla JS
+
+const myElement = document.getElementById('my-element')
+
+myElement.addEventListener('mouseover', () => {
+    posthog.capture('my custom event')
+})
+
+```
+
+Another useful custom event might be tracking exceptions, like so:
+
+```js
+// Vanilla JS
+
+window.onerror = (errorMsg, url, lineNumber) => {
+    // Pass exception details as event properties
+    posthog.capture('exception', {"message": errorMsg, "url": url, "lineNumber": lineNumber})
+    return false;
+}
+```
+
+These exceptions can then be correlated with [feature flags](/docs/features/feature-flags) as well as usage of your app, for example.
+
+Essentially, it's entirely up to you what events you capture and what properties you pass to them. You can also add more complex logic to determine when to trigger an event, as well as what properties to pass to it.
+
+**Example: Custom events from your backend**
```

Comment:
Could we make the example title more specific ie 'Example: Custom events to capture user email addresses from your backend' or similar?